it/uk/gov/hmrc/minorentityidentificationfrontend/assets/TestConstants.scala

```diff
@@ -191,4 +191,10 @@ object TestConstants {
   val testBusinessVerificationFailJson: JsObject = Json.obj(BusinessVerificationStatusKey -> BusinessVerificationFailKey)
   val testBusinessVerificationNotEnoughInfoToChallengeJson: JsObject = Json.obj(BusinessVerificationStatusKey -> BusinessVerificationNotEnoughInfoToChallengeKey)
   val testBusinessVerificationNotEnoughInfoToCallJson: JsObject = Json.obj(BusinessVerificationStatusKey -> BusinessVerificationNotEnoughInfoToCallKey)
+
+  def testTrustKnownFactsJson(correspondencePostcode: String, declarationPostcode: String): JsValue = Json.obj(
+    "correspondencePostcode" -> correspondencePostcode,
+    "declarationPostcode" -> declarationPostcode,
+    "isAbroad" -> false
```

Comment:
Why do we need this when we have testKnownFactsJson above?

Reply:
yes ... you are correct

Reply:
No no Milly we need to talk ... all day call when you can please

Reply:
okay resolved during the chat